docs/source/reference/respawn/player.rst

```diff
@@ -24,7 +36,7 @@ Functions for getting player, and methods of the player object
 
     .. code-block:: javascript
 
-        GetSortedPlayers(int function(entity player1, entity player2) {
+        GetSortedPlayers(function(entity player1, entity player2) {
```

Comment:
Seems like this was in the original too, but i'm fairly sure this has a missing `)`

Reply:
I tested the new and old code and realized that the code before the change was correct. But there are no brackets missing.
Will revert the change later.

Reply:
Oh, I see i actually misread the function declaration. It was correct!